srcpkgs/obs-v4l2sink/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'obs-v4l2sink'
+pkgname=obs-v4l2sink
+version=3.5
+revision=1
+archs="x86_64"
+depends="obs v4l2loopback"
+short_desc="OBS plugin that provides output capabilities to a v4L2 device"
+maintainer="xSlendiX <radu.ungureanu@techie.com>"
+license="GPL-3.0-or-later"
```

Comment:
It's `GPL-2.0 -or-later` https://github.com/CatxFish/obs-v4l2sink/blob/master/src/v4l2sink.cpp